### PR TITLE
Henry autonomous

### PR DESCRIPTION
**Pull request by Prime-235711**
Before issuing a pull request, please see the contributing page.

### CODE REVIEW

**Review by satkab — APPROVED** [review]

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/AutonomousCompetition.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            pauseMillis(750);
+        }
+
+        driveLauncher(0.0);
+
+        switch (ringStackHeight){
+            case 0:
+                driveInches(16,90, 1.0);
+
+                //turnToAngle(45);
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/AutonomousCompetition.java:87]: You shouldn't leave commented out lines in your code.

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterAutonomous.java [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+//
+        //if(targetAngle < startAngle){
+        //    while (imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES).firstAngle - targetAngle <= 1 && opModeIsActive()){
+        //        driveMecanum(0,0,-0.2);
+        //    }
+        //} else{
+        //    while (imu.getAngularOrientation(AxesReference.INTRINSIC, AxesOrder.ZYX, AngleUnit.DEGREES).firstAngle - targetAngle <= 1 && opModeIsActive()){
+        //        driveMecanum(0,0,0.2);
+        //    }
+        //}
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/MasterAutonomous.java:90]: You can delete commented out code

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/MotorTest.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+        //        motorBackRight.setPower(0.0);
+        //    }
+//
+        //    telemetry.addData("FLsp", getMotorTicksPerMinute(motorFrontLeft, 100) / Constants.AM_20_TICKS_PER_ROTATION);
+        //    telemetry.addData("FRsp", getMotorTicksPerMinute(motorFrontRight, 100) / Constants.AM_20_TICKS_PER_ROTATION);
+        //    telemetry.addData("BLsp", getMotorTicksPerMinute(motorBackLeft, 100) / Constants.AM_20_TICKS_PER_ROTATION);
+        //    telemetry.addData("BRsp", getMotorTicksPerMinute(motorBackRight, 100) / Constants.AM_20_TICKS_PER_ROTATION);
+        //   telemetry.addData("FL", motorFrontLeft.getMode());
+        //   telemetry.addData("FR", motorFrontRight.getMode());
+        //   telemetry.addData("BL", motorBackLeft.getMode());
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/MotorTest.java:63]: You can remove all this as well

File: Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/OpenCVTest.java [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+            if(gamepad1.a)
+            {
+                webcam.stopStreaming();
+                webcam.closeCameraDevice();
+            }
+
+            sleep(100);
+
+
```
Comment (malavika1) [inline Team6220_2020/src/main/java/org/firstinspires/ftc/team6220_2020/TestClasses/OpenCVTest.java:72]: Please avoid multiple blank lines

**Review by malavika1 — APPROVED** [review]
In the future, please send out smaller chunks of code to review

**Review by SwerveRoboticsTeams — APPROVED** [review]

**Review by tmicro — APPROVED** [review]

**Pull request merged**